server/testutils/factories/prison.ts

```diff
@@ -2,23 +2,26 @@ import { faker } from '@faker-js/faker/locale/en_GB'
 import { Factory } from 'fishery'
 
 import prisonAddressFactory from './prisonAddress'
-import type { Prison } from '@prison-api'
+import type { Prison } from '@prison-register-api'
 
 export default Factory.define<Prison>(({ params }) => {
-  const locality = params.locality || faker.location.county()
-  const city = params.city || faker.location.city()
-  const country = params.country || 'United Kingdom'
-  const premise = params.premise || `HMP ${locality}`
+  const county = faker.location.county()
+  const prisonName = params.prisonName || `${county} (HMP)`
 
   return {
-    agencyId: faker.string.alpha({ length: 3, casing: 'upper' }),
-    premise,
-    locality,
-    city,
-    country,
-    addresses: [
-      prisonAddressFactory.build({ premise, locality, country, town: city, primary: true }),
-      prisonAddressFactory.build({ premise, primary: false }),
+    prisonId: faker.string.alpha({ length: 3, casing: 'upper' }),
+    prisonName,
+    active: true,
+    male: faker.datatype.boolean(),
+    female: faker.datatype.boolean(),
+    contracted: faker.datatype.boolean(),
+    types: [
+      {
+        code: 'HMP',
+        description: "Her Majesty's Prison",
+      },
     ],
+    addresses: [prisonAddressFactory.build({ county })],
+    operators: [{ name: faker.helpers.arrayElement(['G4S', 'MTCNovo', 'PSP', 'Serco', 'Sodexo']) }],
```

Comment:
Would it be better to use some fake provider names for this (even though the dev data these are from is public)?

Reply:
I don't think it's too much of an issue for the reason you've specified, but let's make these a bit more fake to be safe.